modules/rhcos-enabling-multipath-day-1-power.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bibm_power/installing-ibm-power.adoc
+// * installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc
+
+[id="rhcos-enabling-multipathday-1-power_{context}"]
+= Enabling multipathing with kernel arguments on {op-system}
+
+You can enable multipathing at installation time for nodes that were provisioned in {product-title} 4.9 or later. {op-system} supports multipathing on the primary disk, allowing stronger resilience to hardware failure to achieve higher host availability. Although it is often preferable to modify kernel arguments as a day-2 activity,
+you might want to add kernel arguments to all master or worker nodes during initial cluster
+installation via the machine config.
+
+To add kernel arguments to master or worker nodes, you can create a `MachineConfig` object
```

Comment:
```suggestion
During the initial cluster creation, you might want to add kernel arguments to all master or worker nodes. To add kernel arguments to master or worker nodes, you can create a `MachineConfig` object and inject that object into the set of manifest files used by Ignition during cluster setup. 
```

I would also include a link to where the procedure is for adding kernel arguments. This way you can drop the step to "decide" below.